platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/PlatformConstructor.java

```diff
@@ -207,48 +197,36 @@ static SwirldStateManager swirldStateManager(
     /**
      * Constructs a new {@link PreConsensusEventHandler}.
      *
-     * @param threadManager
-     * 		responsible for creating and managing threads
-     * @param selfId
-     * 		this node's id
-     * @param swirldStateManager
-     * 		the instance of {@link SwirldStateManager}
-     * @param consensusMetrics
-     * 		the class that records stats relating to {@link SwirldStateManager}
+     * @param threadManager      responsible for creating and managing threads
+     * @param selfId             this node's id
+     * @param swirldStateManager the instance of {@link SwirldStateManager}
+     * @param consensusMetrics   the class that records stats relating to {@link SwirldStateManager}
      * @return the newly constructed instance of {@link PreConsensusEventHandler}
      */
     static PreConsensusEventHandler preConsensusEventHandler(
             final ThreadManager threadManager,
             final NodeId selfId,
+            @NonNull final PlatformContext platformContext,
             final SwirldStateManager swirldStateManager,
             final ConsensusMetrics consensusMetrics) {
 
-        return new PreConsensusEventHandler(threadManager, selfId, swirldStateManager, consensusMetrics);
+        return new PreConsensusEventHandler(
+                threadManager, selfId, platformContext, swirldStateManager, consensusMetrics);
     }
 
     /**
      * Constructs a new {@link ConsensusRoundHandler}.
      *
-     * @param threadManager
-     * 		responsible for creating and managing threads
-     * @param selfId
-     * 		this node's id
-     * @param settingsProvider
-     * 		a static settings provider
-     * @param swirldStateManager
-     * 		the instance of {@link SwirldStateManager}
-     * @param consensusHandlingMetrics
-     * 		the class that records stats relating to {@link SwirldStateManager}
-     * @param eventStreamManager
-     * 		the instance that streams consensus events to disk
-     * @param stateHashSignQueue
-     * 		the queue for signed states that need signatures collected
-     * @param enterFreezePeriod
-     * 		a runnable executed when a freeze is entered
-     * @param roundAppliedToStateConsumer
-     * 		the consumer to invoke when a round has just been applied to the state
-     * @param softwareVersion
-     * 		the software version of the application
+     * @param threadManager               responsible for creating and managing threads
```

Comment:
This is not something you changed, but I notice now that this method doesn't have a javadoc for the context... perhaps a good time to add it